packages/video_player/video_player_avfoundation/darwin/Classes/FVPVideoPlayerPlugin.m

```diff
@@ -84,6 +86,7 @@ @interface FVPVideoPlayer ()
 @property(nonatomic, readonly) BOOL isPlaying;
 @property(nonatomic) BOOL isLooping;
 @property(nonatomic, readonly) BOOL isInitialized;
+@property(nonatomic) BOOL isPictureInPictureStarted;
```

Comment:
No `is` for the property, per https://google.github.io/styleguide/objcguide.html#objective-c-method-names